app/assets/stylesheets/application.scss

```diff
@@ -54,6 +54,16 @@ html {
   }
 }
 
+#headerContainer {
+  position: sticky;
+  top: 0;
+  z-index: 1000;
```

Comment:
We want to be deliberate with the `z-index` values we pass in! Let's put the value in `app/assets/stylesheets/base/_values.scss` and use a variable instead! Thanks!